src/dmd/func.d

```diff
@@ -4389,21 +4404,24 @@ private struct AttributeViolation
 /// Params:
 ///   fd = function to check
 ///   maxDepth = up to how many functions deep to report errors
-void errorSupplementalInferredSafety(FuncDeclaration fd, int maxDepth)
+void errorSupplementalInferredSafety(FuncDeclaration fd, int maxDepth, bool deprecation)
```

Comment:
update params

Reply:
There's more DDoc that needs to be added, this is WIP